src/Vision/testpi.py

```diff
@@ -174,8 +174,8 @@ def readCameraConfig(config):
         parseError("could not read height")
         return False
 
-    # stream properties
-    cam.streamConfig = config.get("stream")
+    # modeeam properties
```

Comment:
Something tells me you just did a find-replace `str` -> `mode`... This won't work. It won't get the right config value.